File: Iniciante/URI 1044.ml
let a, b = Scanf.scanf "%i %i" (fun x y -> x, y);;

if (a mod b == 0 || b mod a == 0) then (
    Printf.printf "Sao Multiplos\n";
) else ( 
    Printf.printf "Nao sao Multiplos\n";
)

File: Iniciante/URI 1044.pas
var
    a : int64;
    b : int64;

begin
    readLn(a, b);
	
	if ((a mod b = 0) or (b mod a = 0)) then
		writeln('Sao Multiplos')
	else 
		writeln('Nao sao Multiplos')
end.